### PR TITLE
Add GitHub labeler

### LINKED ISSUE #135: Add GitHub labeler

### PR DESCRIPTION
**Pull request by Rishav1707**
## This PR closes the following issue:-
closes #135

## Changes made:
- Added the pull request labeler

### CODE REVIEW

**Comment by jbampton** [comment]
You also need the file with the labels :)

And then I need to create the labels in this repo

**Comment by Rishav1707** [comment]
> You also need the file with the labels :)
> 
> And then I need to create the labels in this repo

ok

File: .github/workflows/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+jobs:
+  triage:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/labeler@v4
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
```
Comment (jbampton) [inline .github/workflows/labeler.yml:14]:
```suggestion
        repo-token: "${{ secrets.GITHUB_TOKEN }}"

```

File: .github/workflows/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: "Pull Request Labeler"
```
Comment (jbampton) [inline .github/workflows/labeler.yml:1]:
```suggestion
name: Pull Request Labeler
```

**Comment by jbampton** [comment]
Finished in #142

**Comment by jbampton** [comment]
Thank you

**Pull request closed without merging**